docs/development.rst

```diff
@@ -103,44 +109,6 @@ If you don't have a stable branch, create one first or change to that branch::
   git push
 
 
-Setting Up a Local Environment
```

Comment:
I don't think completely removing the development setup instructions is a good step. How to set up the environment will always be fundamentally tied to the version you are working with (e.g. when we switch to pixi those new instructions won't be usable if what you want to work on is an older version that didn't use it yet). Granted, in practice it wouldn't make that much of a difference, since we don't support anything older than the current stable release and will probably try to keep stable and develop in sync in this regard, but if e.g. the switch to pixi would only be done on develop then there would no longer be valid instructions for stable.

As an alternative to removing them, I've modified these instructions for a pixi setup over in https://github.com/Open-MSS/MSS/pull/2298.